truckersmp-cli

```diff
@@ -470,6 +472,30 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_supported_game_versions():
+    """Return a dict of game - version pairs."""
+    # {
+    #     "truckersmp": "0.2.2.6.6",
```

Comment:
I'm not sure whether we really need `truckersmp`. For what purpose can we use this?

Reply:
Removed. I thought it could be useful and didn't hurt but we're indeed already verifying the mod files elsewhere.